examples/Makefile.am

```diff
@@ -45,7 +45,18 @@ VPP_INPUT_SOURCES = \
 	../tests/encodeInputCamera.cpp \
 	$(NULL)
 
-AM_CPPFLAGS = $(AM_CFLAGS)
+AM_CPPFLAGS = $(AM_CFLAGS) \
+	-O2 -D_FORTIFY_SOURCE=2 \
+	-fstack-protector \
+	-fPIC \
+	-fPIE \
+	$(NULL)
+
+AM_LDFLAGS = -z noexecstack \
+	-z relro \
+	-z now \
+	$(NULL)
+
```

Comment:
AM_CPPFLAGS and AM_LDFLAGS shouldn't be here... please follow the same convention as in libyami.